dll/win32/ole32/compobj.c

```diff
@@ -5165,6 +5165,16 @@ HRESULT WINAPI CoRegisterSurrogateEx(REFGUID guid, void *reserved)
     return E_NOTIMPL;
 }
 
+/***********************************************************************
+ *           CoDisconnectContext [OLE32.@]
+ */
+HRESULT WINAPI CoDisconnectContext(DWORD dwTimeout)
```

Comment:
This file in Wine-synced, so these changes would need to be inside a `#ifdef __REACTOS__` block.
But even better, you can use `@ stdcall -stub CoDisconnectContext(long)` in the spec file to do this without any code change at all.

Reply:
If it's not critical i wish to leave it as S_OK stub

Reply:
done! I does not checked that stubs are being generating with returning 0, that equal to S_OK (i wished that). Thank you!